R/cal-validate.R

```diff
@@ -0,0 +1,239 @@
+#' Obtain and validate performance metrics from calibration
+#' @details These functions take an re-sampled object, created via `rsample`,
```

Comment:
Is there a way to take our `tune_results` object and coerce it into the right format to use here? 

In the `tune_results` structure, the `data` object in the elements of `splits` are the training data and these new validate functions requires the predictors to be there. 

We should talk about this in the Details section (that they are not the same) and maybe have a helper function to convert the original results to something that `cal_validate_ *` can consume. 

I don't see a simple way to keep the same resampling structure as the original `tune_results` object.

Reply:
Copied this comment to a new issue (#71).  This will take some thought. I added a note in the Details section